go/libraries/doltcore/sqle/dolt_session.go

```diff
@@ -42,13 +42,15 @@ const (
 
 const EnableTransactionsEnvKey = "DOLT_ENABLE_TRANSACTIONS"
 
-var transactionsEnabled = false
+// TransactionsEnabled controls whether to use SQL transactions
+// Exported only for testing
+var TransactionsEnabled = true
```

Comment:
I think you meant to set this back to `false`. Probably why the test is failing too.

Reply:
yuuuuup